Tests/UnitTests/Core/Utilities/DBScanTests.cpp

```diff
@@ -0,0 +1,106 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2021 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include <boost/test/unit_test.hpp>
+
+#include "Acts/Utilities/DBScan.hpp"
+
+#include <algorithm>
+#include <array>
+#include <cstddef>
+#include <iterator>
+#include <string>
+#include <utility>
+#include <vector>
+
+namespace {
+std::vector<std::array<double, 2>> test_vector{
```

Comment:
Do the values have any special meaning or completely random?
Maybe you could add tiny description like
```
A test vector with random values. They form 4 clusters around ... and should be separable with an epsilon of ...
```

Reply:
I took them from a ML lecture I gave last year. They were generated in python from the sklearn `make_blobs` function to correspond to 4 cluster. I will add a comment